packages/server-common/test/handlers/cookie-preferences/save-cookie-preferences-handler-factory.test.ts

```diff
@@ -0,0 +1,129 @@
+import { DynamoDBClient, PutItemCommand } from '@aws-sdk/client-dynamodb';
+
+import { saveCookiePreferencesHandlerFactory } from '../../../src/handlers/cookie-preferences/save-cookie-preferences-handler-factory';
+import { Logger } from '../../../src/utils';
+import { getLambdaRequest } from '../../helpers/events';
+
+jest.mock('@aws-sdk/client-dynamodb');
+
+const logger = {
+  info: jest.fn(),
+  debug: jest.fn(),
+} as unknown as jest.Mocked<Logger>;
+
+describe('Save cookie preferences handler', () => {
+  const tableName = 'cookie-preferences-table';
+  const handler = saveCookiePreferencesHandlerFactory(logger, tableName);
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('Validations', () => {
+    test('returns 400 when cookieId is not defined', async () => {
+      const request = getLambdaRequest({ preferences: {} }, {});
+      expect(handler(request)).rejects.toThrow('Validation error');
+    });
+
+    test('returns 400 when preferences is not defined', async () => {
```

Comment:
It looks like we're using both `test` and `it`. Can we pick one and stick with it please?